config/routes.rb

```diff
@@ -7,6 +7,7 @@
 
   resources :tasks
 
+  post '/twilio_action' => 'tasks#twilio_action'
```

Comment:
I think you can add this to your `tasks` resources above:

```ruby
resources :tasks do
  post :twilio_action, on: :collection
end
```

Check this out for reference: http://guides.rubyonrails.org/routing.html#adding-more-restful-actions